apps/condo/domains/banking/schema/BankTransaction.js

```diff
@@ -0,0 +1,115 @@
+const access = require('@condo/domains/banking/access/BankTransaction')
+const { ORGANIZATION_OWNED_FIELD } = require('../../organization/schema/fields')
+const { MONEY_AMOUNT_FIELD } = require('@condo/domains/common/schema/fields')
+const { CURRENCY_CODE_FIELD } = require('../../common/schema/fields')
+
+
+const BankTransaction = new GQLListSchema('BankTransaction', {
+    schemaDoc: 'Transaction related to costs of Organization with BankAccount. Full fields set from data import will be stored in "meta"',
+    fields: {
+
+        account: {
+            schemaDoc: 'Related BankAccount of Organization which payed ',
+            type: Relationship,
+            ref: 'BankAccount',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        contractorAccount: {
+            schemaDoc: 'Related account of contractor, which has received the payment via this transaction',
+            type: Relationship,
+            ref: 'BankContractorAccount',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        costItem: {
+            schemaDoc: 'Related costs class',
+            type: Relationship,
+            ref: 'BankCostItem',
+            kmigratorOptions: { null: true, on_delete: 'models.SET_NULL' },
+        },
+
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        number: {
+            schemaDoc: 'Number of transaction, obtained from external system',
+            type: Text,
+            isRequired: true,
+        },
+
+        date: {
+            schemaDoc: 'When payment order was created',
+            type: CalendarDay,
+            isRequired: true,
+        },
+
+        amount: {
+            ...MONEY_AMOUNT_FIELD,
+            schemaDoc: 'Amount of transaction in specified currency',
+            isRequired: true,
+        },
+
+        currencyCode: CURRENCY_CODE_FIELD,
+
+        purpose: {
+            schemaDoc: 'Textual description of payment purpose in free form',
+            type: Text,
+            isRequired: true,
+        },
+
+        dateWithdrawed: {
+            schemaDoc: 'Date when this payment was withdrawed from BankAccount of Organization. If filled, then this payment is an outcome',
+            type: CalendarDay,
+            isRequired: true,
+        },
+
+        dateReceived: {
+            schemaDoc: 'Date when this payment was received by BankAccount of Organization. If filled, then this payment is an income',
+            type: CalendarDay,
+            isRequired: true,
+        },
+
+        meta: {
+            schemaDoc: 'Stores data, obtained from external source',
+            type: Json,
+            isRequired: true,
```

Comment:
Is it possible to specify types for meta and add `ajv` validation?

Reply:
In `meta` field raw data from data sources will be stored.
Because it will be stored as is, I don't see a reason for data validation.
Different data sources will store it's data using namespaced keys.

Example for raw data from exported file from 1C:

```json
{
    meta: {
        '1CClientBankExchange': {
            'v': '1.03',
            'data': {
            }
        }
    }
}
```

The same will be for raw data from SBBOL:

```json

```json
{
    meta: {
        'SBBOL': {
            'v': '…',
            'data': {
            }
        }
    }
}
```